src/components/smartSearch/utils.ts

```diff
@@ -0,0 +1,43 @@
+import { TIME_FRAME } from 'types/smartSearch';
+
+interface timeFrameConfig {
```

Comment:
Maybe I'm mistaken but I think we consistently use UpperCamelCase ([a.k.a. `PascalCase`](https://en.wikipedia.org/wiki/Camel_case#Variations_and_synonyms)) for interfaces and other types, and probably should be here as well.